spec/std/string_spec.cr

```diff
@@ -188,6 +188,16 @@ describe "String" do
       "hello"[1, 3]?.should eq("ell")
       "hello"[6, 3]?.should be_nil
     end
+
+    it "gets with range without end" do
+      "há日本語"[1..nil]?.should eq("á日本語")
+      "hello"[6..nil]?.should be_nil
+    end
+
+    it "gets with range without beginning" do
+      "há日本語"[nil..2]?.should eq("há日")
+      "hello"[nil..6]?.should be_nil
```

Comment:
@asterite Shouldn't this line return a nil and throw exception without `?` option? Another bug? Thoughts?

```cr
"hello"[nil..7] # this doesn't raise error
"hello"[6..7] # this raises an error
```

https://play.crystal-lang.org/#/r/85o1